java/arcs/android/type/ParcelableSchemaFields.kt

```diff
@@ -122,6 +135,10 @@ data class ParcelableSchemaFields(val actual: SchemaFields) : Parcelable {
           }
         }
       }
+      is FieldType.NullableOf -> {
+        writeByte('?'.toByte())
```

Comment:
Is this byte necessary? I would have though the type tag written in line 112 would have been sufficient for identifying the field type. Plus it doesn't look like you actually read it back out again.

Reply:
Regardless, can you update the unit tests for this file?

Reply:
Yup, just misunderstood the code from Tuple ( we should be able to remove some of the bytes there actually some time (i.e. I don't think we need the separators or opening paren)

Reply:
Removed the byte and updated unit tests.